rocky/reports/views/report_overview.py

```diff
@@ -237,14 +243,26 @@ def rerun_reports(self, report_references: list[str]) -> None:
                     ),
                 )
             else:
-                self.rerun_report(report_ooi)
-
-                for asset_report in report_ooi.input_oois:
-                    self.rerun_report(asset_report)
-
-        messages.success(
-            self.request, _("Rerun successful. It may take a moment before the new report has been generated.")
-        )
+                updated_all = self.rerun_report(report_ooi)
+
+                if updated_all:
+                    for asset_report in report_ooi.input_oois:
+                        if not self.rerun_report(asset_report):
+                            updated_all = False
+                if not updated_all:
+                    not_updated_reports.append(report_ooi.name)
+        if not not_updated_reports:
+            messages.success(
+                self.request, _("Rerun successful. It may take a moment before the new report has been generated.")
+            )
+        else:
+            messages.warning(
+                self.request,
+                _(
+                    f"Couldn't rerun { ', '.join(not_updated_reports) }, since the recipe for this "
+                    "report has been disabled or deleted."
+                ),
```

Comment:
You shouldn't use reformatted strings when translating. This will result in untranslatable lines and it's inneficient. See also the .pot file